src/cmd-compress

```diff
@@ -95,7 +95,9 @@ def compress_one_builddir(builddir):
         img = buildmeta['images'][img_format]
         file = img['path']
         filepath = os.path.join(builddir, file)
-        if not file.endswith(ext):
+        # check if it's not already compressed, even if with a different
+        # compressor
```

Comment:
Might be worth adding. *This prevents us from getting a tar.gz.xz like we were for GCP images*

Or we could just link to the PR.

Reply:
Added link!